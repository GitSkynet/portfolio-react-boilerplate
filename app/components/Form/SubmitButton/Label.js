import styled from 'styled-components';

const Label = styled.span`
  margin-left: 0.5em;
`;

export default Label;
