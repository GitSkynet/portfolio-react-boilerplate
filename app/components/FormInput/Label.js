import styled from 'styled-components';

const Label = styled.label`
  position: relative;
  display: inline-block;
  width: 100%;
  font-size: 16px;
  height: 12px; // font size on focus
`;

export default Label;
