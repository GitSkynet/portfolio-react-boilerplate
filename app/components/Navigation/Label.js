import styled from 'styled-components';

const Label = styled.span`
  margin-left: 0.5rem;
`;

export default Label;
