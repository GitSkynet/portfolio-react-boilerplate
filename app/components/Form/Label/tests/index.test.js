import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import Label from '../index';

describe('<Label />', () => {
  it('should render and match the snapshot', () => {
    const renderedComponent = shallow(<Label />);
    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
