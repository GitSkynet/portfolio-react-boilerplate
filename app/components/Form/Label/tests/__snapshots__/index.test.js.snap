// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> should render and match the snapshot 1`] = `
.c0 {
  position: relative;
  display: inline-block;
  width: 100%;
  font-size: 16px;
  height: 12px;
}

<label
  className="c0"
/>
`;
